examples/gaussian_process/plot_gpr_dim_selection.py

```diff
@@ -0,0 +1,101 @@
+"""
+=========================================================
+Illustration of SelectDimensionKernel
+=========================================================
+A simple two-dimensional regression example computed in two different ways:
+1. With a product of two RBF kernels on each feature.
```

Comment:
It should be product I guess since RBF kernel is defined as:

k(x_i, x_j) = exp(-1 / 2 d(x_i / length_scale, x_j / length_scale)^2)

[here](http://scikit-learn.org/dev/modules/generated/sklearn.gaussian_process.kernels.RBF.html#sklearn.gaussian_process.kernels.RBF) and therefore product of two RBFs would produce the sum of two quadratics.